message/validation/validation.go

```diff
@@ -101,6 +110,8 @@ func NewMessageValidator(netCfg networkconfig.NetworkConfig, opts ...Option) Mes
 		opt(mv)
 	}
 
+	go mv.state.Start()
```

Comment:
This should move to a `Start` function of message validation, and `Stop` should also be added on a stop function.